modules/ztp-worker-node-preparing-policies.adoc

```diff
@@ -0,0 +1,138 @@
+              user:
+                name: root
+    - fileName: PerformanceProfile.yaml
+      policyName: "config-policy"
+      metadata:
+        name: openshift-worker-node-performance-profile
+      spec:
+        cpu: <4>
+          isolated: "4-47"
+          reserved: "0-3"
+        hugepages:
+          defaultHugepagesSize: 1G
+          pages:
+            - size: 1G
+              count: 32
+    - fileName: TunedPerformancePatch.yaml
+      policyName: "config-policy"
+      metadata:
+        name: performance-patch-worker
+      spec:
+        profile:
+          - name: performance-patch-worker
+            data: |
+              [main]
+              summary=Configuration changes profile inherited from performance created tuned
+              include=openshift-node-performance-openshift-worker-node-performance-profile
+              [bootloader]
+              cmdline_crash=nohz_full=4-47 <5>
+              [sysctl]
+              kernel.timer_migration=1
+              [scheduler]
+              group.ice-ptp=0:f:10:*:ice-ptp.*
+              [service]
+              service.stalld=start,enable
+              service.chronyd=stop,disable
+        recommend:
+        - profile: performance-patch-worker
+    - fileName: PtpOperatorConfig.yaml
+      policyName: "config-policy"
+      complianceType: mustonlyhave
+      spec:
+        daemonNodeSelector:
+          node-role.kubernetes.io/worker: ""
+    - fileName: SriovOperatorConfig.yaml
+      policyName: "sriov-config-policy"
+      complianceType: mustonlyhave
+      spec:
+        configDaemonNodeSelector:
+          node-role.kubernetes.io/worker: ""
+----
+<1> The policies are applied to all clusters with this label.
+<2> The `MCP` field must be set to `worker`.
+<3> This generic `MachineConfig` CR is used to configure workload partitioning on the worker node.
+<4> The `cpu.isolated` and `cpu.reserved` fields must be configured for each particular hardware platform.
+<5> The `cmdline_crash` CPU set must match the `cpu.isolated` set in the `PerformanceProfile` section.
+
++
+A generic `MachineConfig` CR is used to configure workload partitioning on the worker node. You can generate the content of `crio` and `kubelet` configuration files.
+
+. Add the created policy template to the Git repository monitored by the ArgoCD `policies` application.
+
+. Add the policy in the `kustomization.yaml` file.
+
+. Commit the changes in Git, and then push to the Git repository being monitored by the GitOps ZTP ArgoCD application.
+
+. To remediate the new policies to your spoke cluster, create a TALM custom resource:
+
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: example-sno-worker-policies
+  namespace: default
+spec:
+  backup: false
+  clusters:
+  - example-sno
+  enable: true
+  managedPolicies:
+  - example-sno-workers-config-policy
+  - example-sno-config-policy
```

Comment:
We did not document modifying existing ptp (in group pgt) and sriov (in sno pgt) configs, which is why this line is needed in CGU.  Need to add documentation modifying existing ptp and sriov configs.

Reply:
We documented it in the [PTP and SR-IOV node selector compatibility](https://github.com/openshift/openshift-docs/pull/50251/files#diff-3bae2051f74457a2906a6e3c35a473dc44ebe2bc7fe15622b6984ae0106c59b9)
Line 9

Reply:
@yliu127 Isn't modifying existing configs documented [here](https://50251--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-sno-additional-worker-node.html#ztp-additional-worker-node-selector-comp_sno-additional-worker) and (with optional modification) [here](https://50251--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-sno-additional-worker-node.html#ztp-additional-worker-daemon-selector-comp_sno-additional-worker)?

Reply:
@amolnar-rh  somehow your links just bring me to the top of the html doc. So not too sure which sections you are referring to. But as @vitus133  pointed out, there is indeed a sentence to briefly describe what needs to be done. So I'm okay with this.